File: 06Adapter/Students/2018/DuniaWojciech/TwoWayAdapter/iaircraft.py
from abc import ABC, abstractmethod


class IAirCraft(ABC):
    @abstractmethod
    def take_off(self):
        pass

    @property
    def airborne(self):
        return self._airborne

    @airborne.setter
    def airborne(self, value):
        self._airborne = value

    @property
    def height(self):
        return self._height

    @height.setter
    def height(self, value):
        self._height = value

File: 06Adapter/Students/2019/ZielińskiŁukasz/IAircraft.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Adapter {
    // ITarget interface
    public interface IAircraft {
        bool Airborne { get; }
        void TakeOff();
        int Height { get; set; }
    }
}
